### DISCUSSION TITLE
Welcome to home-connect-hass Discussions!

### DISCUSSION THREAD

**Discussion by ekutner** [ekutner/home-connect-hass#74] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by von-Chaps** [comment]
Hi there and thanks for setting up the discussion. I  hope it has the visibility so that it's discoverable by people.

I've used HASS for a number of years now and have it pretty much set up how I like it. I'm in the UK so I make a lot of use of Hive. I just put a new kitchen in and ended up with a Bosch oven with HC, so here I am looking to understand the integration and to make good use of it.

I've a bunch of questions about how this integration works - which I'll put in the Q&A section, but I'm also interested in how people use it. I am thinking I'd like to set up some automations with favourite settings. Like push a button in HA and have the oven come on with Pizza Setting for 30 mins at 200, starting in 10 mins. It's all doable from the HC app (with favourites), but it would be easier with some preset automations in HA. I'd be interested to hear what other people are doing with this integration.

I'll also offer to help with the development of the integration if that's required - even just testing or docs. I'd like to put something back. I was a s/w dev and I know it can be nice not to work alone all the time.

**Comment by ekutner** [comment]
Welcome and any input/contribution will be appreciated.
If you wish to contribute some code then right now the only item on my to-do list is related to the issue you reported with the timer. I'd like to have a time selection control for time selection options/settings so users don't have to enter the number of seconds, which is not very friendly. I created a custom select box for the delayed start option but that was when I thought there was only one instance of such an entity. I'd like this to be generalized, and preferably not use a select but a real time selection control. If you have some time, take a look at that.
